### DISCUSSION TITLE
eCAL will move to Eclipse

### DISCUSSION THREAD

**Discussion by FlorianReimold** [eclipse-ecal/ecal#724] [Announcements]
If everything goes to plan, on Thursday, 2022-07-28 (-> Tomorrow 😉) this repository will become an Eclipse Project and therefore be moved to a new GitHub Organization:

https://github.com/eclipse-ecal

The old location at /continental/ will be replaced by a fork, so existing links will stay valid. We will try to fix everything that breaks as fast as possible, though.

**Comment by ahoarau** [comment]
What does this means in terms of maintainers ? Continental is no longer financing this ?

**Reply by rex-schilasky** [reply]
Financing is fine in the meaning of Continental's engineering is still investing the same effort even eCAL moved under the umbrella of Eclipse. It's just the repository management and the workflow that is now done under Eclipse Foundation. Finally we want to integrate eCAL in an ecosystem of software solutions for Software Defined Vehicles (https://sdv.eclipse.org/)
